olive/passes/onnx/quantization.py

```diff
@@ -530,7 +530,7 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
             ),
             "nodes_to_exclude": PassConfigParam(
                 type_=list,
-                default_value=[],
+                default_value=None,
```

Comment:
Double check: in this case `[]` should be fine?

Reply:
`[]` also works but since we don't use `[]` as default value in other parts of the code base, I changed it to None. The quantizer can also handle None.